
=========================================================================================================================== 

ERROR OCCURRED

DATE & TIME: 26-01-2018 8.09.02 PM
SOURCE: Microsoft.EntityFrameworkCore.Relational
METHOD: Void Execute(Microsoft.EntityFrameworkCore.Storage.IRelationalConnection)
ERROR: An error occurred while updating the entries. See the inner exception for details.
STACKTRACE:    at Microsoft.EntityFrameworkCore.Update.ReaderModificationCommandBatch.Execute(IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Update.Internal.MySqlBatchExecutor.Execute(IEnumerable`1 commandBatches, IRelationalConnection connection)
   at Microsoft.EntityFrameworkCore.Storage.RelationalDatabase.SaveChanges(IReadOnlyList`1 entries)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(IReadOnlyList`1 entriesToSave)
   at Microsoft.EntityFrameworkCore.ChangeTracking.Internal.StateManager.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges(Boolean acceptAllChangesOnSuccess)
   at Microsoft.EntityFrameworkCore.DbContext.SaveChanges()
   at PIT.BAL.Services.Dictonary.DictonaryService.Insert(DictonaryModel obj) in E:\CSharpProgram\gitProject\PixInfotechProject\Dictionary\API\PIT.BAL\Services\Dictonary\DictonaryService.cs:line 125
